crates/networking/rpc/clients/eth/mod.rs

```diff
@@ -100,24 +104,35 @@ pub struct WithdrawalProof {
 
 impl EthClient {
     pub fn new(url: &str) -> Self {
-        Self {
-            client: Client::new(),
-            url: url.to_string(),
-            maximum_allowed_max_fee_per_gas: None,
-            maximum_allowed_max_fee_per_blob_gas: None,
-        }
+        Self::new_with_config(
+            url,
+            MAX_NUMBER_OF_RETRIES,
+            BACKOFF_FACTOR,
+            MIN_RETRY_DELAY,
+            MAX_RETRY_DELAY,
+            None,
+            None,
+        )
```

Comment:
After doing the suggested refactor, implement `Default` for `EthConfig` with these values and use `EthConfig::default()` here.

Reply:
This cannot have a default since the `url` is not default. We also cannot import EthConfig because of a circular dependency issue.